contracts/command-batcher/src/encoding/evm.rs

```diff
@@ -0,0 +1,557 @@
+use std::{collections::HashMap, fmt::Display, str::FromStr};
+
+use axelar_wasm_std::Snapshot;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{HexBinary, Uint256};
+use ethabi::{ethereum_types, Token};
+use k256::{elliptic_curve::sec1::ToEncodedPoint, PublicKey};
+use multisig::types::Signature;
+use sha3::{Digest, Keccak256};
+
+use crate::{
+    error::ContractError,
+    types::{CommandBatch, Operator, Proof},
+};
+
+use super::traits;
+
+#[derive(Debug)]
+pub struct Message {
+    pub id: String,
+    pub source_address: String,
+    pub source_chain: String,
+    pub destination_address: ethereum_types::Address,
+    pub payload_hash: [u8; 32],
+}
```

Comment:
What do we need this type for?

Reply:
you are right it's not needed, it was used before as intermediary struct for creating validated input, but then I added the type `Command` and it's easier to just go directly from router `Message` to `Command` since it's the only place where is needed. Removed